packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -173,7 +173,10 @@ export class PipelineEditor extends React.Component<
     this.canvasController.setPipelineFlowPalette(palette);
     this.widgetContext = props.widgetContext;
     this.widgetContext.ready.then(() => {
-      this.canvasController.setPipelineFlow(this.widgetContext.model.toJSON());
+      const pipelineFlow = PipelineService.convertPipeline(
```

Comment:
Kinda thinking this should just be called `openPipeline()` and let this method determine if conversion should occur or not.  Otherwise, it seems strange that the very first thing done on a pipeline is to convert when 99% of the time, nothing will be converted.

Reply:
I was on the fence over this, as this is really only handling the "read and display" of the pipeline file, which to me was load, while open was more the whole opening widget thing. But at the end I have renamed it.